nixos/modules/system/boot/stage-1-init.sh

```diff
@@ -222,6 +229,9 @@ for o in $(cat /proc/cmdline); do
         boot.panic_on_fail|stage1panic=1)
             panicOnFail=1
             ;;
+        nixos.rekey)
```

Comment:
Do we really want this? Looks like a feature creeping in that might be hard/annoying to get rid of again.

Reply:
you're right